[ET-1078] Blocks > Fix RSVP & Tickets block description (#1981)
the-events-calendar/event-tickets: readme.txt

= [5.1.3] TBD =

> * Fix - Fixed an issue where long descriptions were breaking the block for the Tickets and RSVP blocks. It's now using auto-size textarea. [ET-1078]

Reviewer: ```suggestion
* Fix - Fixed an issue with Tickets and RSVP blocks where long descriptions were breaking the block. They now use an auto-resizing textarea. [ET-1078]
```
